tests/eclient/testdata/dev_local_info.txt

```diff
@@ -0,0 +1,241 @@
+stdout 'app1'
+! stderr .
+
+
+# STEP 9: Remove the second app
+eden pod delete app1
+test eden.app.test -test.v -timewait 15m - app1
+
+# STEP 10: Undeploy local-manager
+eden pod delete local-manager
+test eden.app.test -test.v -timewait 15m - local-manager
+eden network delete {{template "network"}}
+test eden.network.test -test.v -timewait 10m - {{template "network"}}
+
+-- wait-ssh.sh --
+EDEN={{EdenConfig "eden.root"}}/{{EdenConfig "eden.bin-dist"}}/{{EdenConfig "eden.eden-bin"}}
+HOST=$($EDEN eve ip)
+for p in $*
+do
+  for i in `seq 20`
+  do
+    sleep 20
+    # Test SSH-access to container
+    echo {{template "ssh"}}$HOST -p $p grep -q Ubuntu /etc/issue
+    {{template "ssh"}}$HOST -p $p grep -q Ubuntu /etc/issue && break
+  done
+done
+
+-- local-manager-start.sh --
+EDEN={{EdenConfig "eden.root"}}/{{EdenConfig "eden.bin-dist"}}/{{EdenConfig "eden.eden-bin"}}
+HOST=$($EDEN eve ip)
+ARGS="--token={{template "token"}}"
+{{template "ssh"}}$HOST -p {{template "mngr_port"}} "/root/local_manager $ARGS &>/dev/null &"
+
+-- get-app-ip.sh --
+EDEN={{EdenConfig "eden.root"}}/{{EdenConfig "eden.bin-dist"}}/{{EdenConfig "eden.eden-bin"}}
+IP=$($EDEN pod ps | grep $1 | awk '{print $4}' | cut -d ":" -f 1)
+echo app_ip=$IP>>.env
+
+-- get-appinfo-status.sh --
+EDEN={{EdenConfig "eden.root"}}/{{EdenConfig "eden.bin-dist"}}/{{EdenConfig "eden.eden-bin"}}
+HOST=$($EDEN eve ip)
+APP="$1"
+CMDS="
+until test -f {{template "app_info_status_file"}}; do sleep 5; done
+sleep 2
+cat {{template "app_info_status_file"}}
+"
+
+OUTPUT="$({{template "ssh"}}$HOST -p {{template "mngr_port"}} "$CMDS")"
+if [ -n "$APP" ]; then
+    echo "$OUTPUT" | jq --arg APP "$APP" '.appsInfo[] | select(.name==$APP)'
+else
+    echo "$OUTPUT"
+fi
+
+-- wait-for-app-state.sh --
+APP="${1}"
+EXPSTATE="${2}"
+
+EDEN={{EdenConfig "eden.root"}}/{{EdenConfig "eden.bin-dist"}}/{{EdenConfig "eden.eden-bin"}}
+HOST=$($EDEN eve ip)
+
+while true; do
+    APPINFO="$({{template "ssh"}}$HOST -p {{template "mngr_port"}} "cat {{template "app_info_status_file"}}")"
+    APPINFO="$(echo "$APPINFO" | jq --arg APP "$APP" '.appsInfo[] | select(.name==$APP)')"
+    echo "$APPINFO" | grep "$EXPSTATE" && break
+    sleep 1
+done
+
+-- get-devinfo-status.sh --
+EDEN={{EdenConfig "eden.root"}}/{{EdenConfig "eden.bin-dist"}}/{{EdenConfig "eden.eden-bin"}}
+HOST=$($EDEN eve ip)
+CMDS="
+until test -f {{template "dev_info_status_file"}}; do sleep 5; done
+sleep 2
+cat {{template "dev_info_status_file"}}
+"
+
+OUTPUT="$({{template "ssh"}}$HOST -p {{template "mngr_port"}} "$CMDS")"
+echo "$OUTPUT"
+
+-- put-devinfo-cmd.sh --
+DN="$1"
+TIMESTAMP="${2:-0}"
+CMD="${3:-COMMAND_UNSPECIFIED}"
+
+EDEN={{EdenConfig "eden.root"}}/{{EdenConfig "eden.bin-dist"}}/{{EdenConfig "eden.eden-bin"}}
+HOST=$($EDEN eve ip)
+CONFIG="{\"displayname\": \"$DN\", \"timestamp\": $TIMESTAMP, \"command\": \"$CMD\"}"
+echo "$CONFIG" | {{template "ssh"}}$HOST -p {{template "mngr_port"}} 'cat > {{template "dev_cmd_file"}}'
```

Comment:
As I understand it should be json representation of [LocalDevCmd](https://github.com/lf-edge/eve/blob/master/api/go/profile/local_profile.pb.go#L815-L826), so command is enough. Also I can see only one argument for calling this function, so please remove DN and TIMESTAMP.

Reply:
Timestamp set automatically based on file modification time here: https://github.com/lf-edge/eden/blob/99d7f6244a1b98b3fb1b3372f0346bb44babebe1/tests/eclient/image/pkg/main.go#L219

Reply:
Right. Fixing that.